src/app/Components/login/login.component.html

```diff
@@ -0,0 +1,30 @@
+
+  <form class="login-form">
+    <mat-form-field class="login-full-width">
+      <mat-label>Email*</mat-label>
+      <input type="email" matInput [formControl]="emailFormControl" placeholder="Ex. pat@example.com" />
+      <mat-error *ngIf="emailFormControl.hasError('email')">
+        Please enter a valid email address
+      </mat-error>
+      <mat-error *ngIf="emailFormControl.hasError('required')">
+        Email is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field class="login-full-width">
+      <mat-label>Password*</mat-label>
+      <input class="form-control" matInput [type]="hide ? 'password' : 'text'" [formControl]="passwordFormControl"
+        placeholder="Password" />
+
+      <mat-error *ngIf="passwordFormControl.hasError('required')">
+        <strong>Password required</strong>
```

Comment:
[here](https://github.com/simonaVasilachi/my-personal-project/pull/7/files#diff-3e083c2dc0df58a12e57d556c91104da4e5a81b4e8e2789a4607ff5e1ad361c6R10) you apply `storng` only to a partial text. 
you should decide on consistency